### ISSUE TITLE
Philosophy of Architecture (Stanford Encyclopedia of Philosophy)

### ISSUE THREAD

**Issue by occoder** [occoder/common_sql#200]
1\. Introduction
----------------

### 1.1 Architecture as Relatively Neglected by Philosophy

Over the course of Western philosophy, including the history of aesthetics, architecture has largely failed to attract sustained, detailed attention—particularly as compared with other artforms. Neither philosophical issues prompted by architecture, nor the fit of architectural phenomena into larger philosophical debates, have captured the philosophical imagination as have, for example, literature or painting. Some contributions across the span of Western philosophy—including those of Wolff and Schopenhauer—rank as historically significant; other, more recent accounts are broad-ranging and gravid with conceptual concerns, including those of Scruton and Harries. Further, some philosophers have even dabbled in architectural projects: Dewey contributed to plans for the Chicago Laboratory School, Wittgenstein collaborated on designing a house for his sister, and Bentham sketched the Panopticon design as a plan for prison reform. Yet the overall state of philosophical reflection on architecture—even in the present day—is less lively than like discussions focused on artforms of far more recent origin, such as film or comics. Some philosophers working in the Continental tradition have offered accounts of the experience of architecture or its social ramifications; a deficiency is more marked in analytic aesthetics.

For more on the background and context of conceptual explorations of architecture, see the supplementary documents:

> [Philosophy of Architecture in Historical Perspective](https://plato.stanford.edu/entries/architecture/perspective.html)  
> [Philosophy and the Tradition of Architectural Theory](https://plato.stanford.edu/entries/architecture/tradition.html)

### 1.2 A Word on Terminology

This essay refers generally to the basic creative output of architects, in any (unspecified) form, as “architectural objects”. This is in parallel with the term “art objects” in use, across aesthetics and philosophy of art, to refer to objects created by artists independent of the artform and without regard to ontological or other discussions where other terms may evoke one or another particular stance. It may turn out that “architectural object” proves value-laden. Yet this is less likely than with common alternatives in the literature: “architectural works”, perhaps wedded to creator intent, status as extant or integral whole, or aesthetic ranking; or “buildings”, certainly wedded to built, hence concrete, structures. “Built structures” is used here (except in historical reference) to refer to the generic built output resulting from architectural design, in recognition of a domain some view as broader than buildings _per se_.

2\. What is Architecture?
-------------------------

Fundamental questions about the nature of architecture motivate much of contemporary philosophy of architecture: what sort of enterprise architecture is; whether architecture has essential features; what kinds of things architecture makes—yielding the further issue as to whether architecture always, only sometimes, or never is an artform; what renders architecture distinct from other artforms (if it is one); and whether architecture includes all built structures.

### 2.1 The Sort of Enterprise Architecture Is

One approach to grasping the true nature of architecture is to define it in terms of the discipline. We may embrace the disciplinary determinism of the British architect Cedric Price: “Architecture is what architects do”. Defining the discipline or practice of architecture may seem a simple empirical affair. Even if we have difficulty assessing what architectural objects or products are, we can point to thousands of architects over several millennia around the world engaged in one or another sets of activities that conventionally have been associated with architectural practice, and generate a long disjunctive claim about what architects do. An empirically rooted approach has a long history: Vitruvius devises his normative account of the virtuous architect on the basis of his familiarity with then-contemporary practice. The same is true of other traditional accounts of virtue in architects or builders, as in the sixth to eighth century Indian _Mānasāra_ (मानसार) (Acharya 1928) and eleventh century Chinese _Yingzao fashi_ (營造法式) (Feng 2012). In the present-day, sociology of the architectural profession offers a detailed empirical perspective (Gutman 1988), and this can be extended to a sociology of architectural worlds, modeled on Becker’s sociology of art worlds.

A problem arises, though, if we look to history or sociology for a unified account with common features of an architectural discipline. While architectural practice has remained stable in certain respects, change over its history greatly limits common features, perhaps, to a core set of basic tools and rudimentary principles of structural engineering. This suggests that, at root, the practice of architecture must involve engineering or related design. But architecture can’t be reduced to a form of engineering, if we think architectural ideals, taste, and expertise contribute something over and above engineering facts, rules, and practical knowledge. These further contributions suggest an art or art-like role for architectural practice. However, the historical record is mixed on the matter of whether architects are at the same time pursuing art (or what we now consider as art) or should be thought of as artists. Moreover, the historical record and resulting disjunctive claim do not address cases where even the most basic tools or structural principles are not deployed. Some such cases of not observing basic structural principles, as fantasy architecture, may be deemed marginal; other such cases, as landscape architecture, are not.

Another dimension of defining architecture as a practice is specifying the sorts of structures that architects design. At a bare minimum, we can say that they feature some connection to human use. But attempts to render this as non-trivial introduce further puzzles ([see §2.3](#KinThiArcMak)).

### 2.2 Architecture and Essential Features

A contrasting definitional approach suggests that, as a matter of reasoned judgment, we can attribute to architectural practice—or to the domain of architectural objects—core or even essential features. A dominant reading of the Vitruvian tradition has it that architecture embodies and is best understood through the three aspects of beauty, structural integrity, and utility. An essentialist variant suggests that architects must observe all three aspects or that any structure aspiring to architectural status features all three. Other prominent views advance a single aspect, generally function or form, as primary. Thus, functionalist architectural doctrine places function or utility at the heart of the architectural enterprise, with other aspects of architecture subordinate thereto.

A hard-line functional essentialist holds that, if a built structure has no function, then it is not architecture. As a modest dissent, Graham (1989) proposes that such a structure _is_ an architectural work—but a _failure_ at such. One brand of more radical rejection suggests that some architectural objects—perhaps including follies, memorials, or monuments—need have no function at all. As a competing essentialism, formalist architectural doctrine suggests that an object is architectural just in case it features forms proper to the domain. A common interpretation says that forms proper to architecture can be chosen off a stylistic menu (or combination of menus), leaving architects great latitude while upholding the possibility of contrasting, non-architectural forms (this is difficult to square, however, with some experimental architecture).

In weighting architecture’s aspects as essential, core, or some lesser status, a related question is whether one or another aspect is primary or necessary to any of the others. As Graham notes (1989), the traditional question in architectural theory of whether form trumps or precedes function may be cast in such terms.

Against these traditional brands of essentialism, two further kinds of doubt may be cast. First, it may be that the Vitruvian triad, or some single aspect thereof, does not represent the right list—we should include either further aspects or different aspects altogether. Alternatives might include dimensions such as context, relations among architectural objects, systemic features, sustainability, and psychological or social features. Some theorists propose other candidates as essential architectural aspects, including space (Zevi 1978) or the organizing concept of the _parti_ (Malo 1999).

Second, it may be that essentialism represents a false start. On one non-essentialist view, the nature of an architectural object is as we experience it, a matter upon which we may agree but which depends in part on subjective perception and reception of, and interaction with, the work (Scruton 1979/2013). This leaves open the possibility that architecture has essential aspects but we simply don’t experience them as such. A more determined nominalist has it that diversity among architectural objects is sufficient to quash the prospect that they share any _essential_ aspects.

### 2.3 The Kinds of Things Architecture Makes

Yet another way to pose the question of what architecture is focuses on the sorts of things architectural objects are. In particular, and relative to architecture’s possible status as artform, architecture as a domain may be defined variously in terms of its objects being art objects (or not), being distinctive sorts of art objects, or belonging exhaustively to a _special_ class of built structures (rather than including all such structures).

**Whether architecture always, only sometimes, or never is an artform.** At the negative extreme, architecture may be viewed like any engineered artifact that only incidentally bears aesthetic value. Any view of even slightly more positive valence bows in the direction of intent to generate aesthetic value. The classic Vitruvian view, for example, has it that engineered design and aesthetic design are conjoint intentional elements of architectural objects. At the positive extreme, that is, the suggestion that architecture is always and in all ways an art, we may lose any means of discriminating among built structures as art or not. This is a troubling prospect to exclusivists who see architecture as a high art only (see below).

In the negative camp, S. Davies (1994) argues that mere production of occasional artworks does not suffice to constitute an artform—crafts being a notable example—and the aim of architecture is frequently, or even typically, not the production of art but useful items that do not aim at artistic value. In the positive camp, Stecker (2010) responds that we can carve out a subclass of architectural objects that are art even if not all are. He adds, by way of historical argument, that architecture was included among the artforms, by early agreement among aestheticians. We might have grounds for dismissing architecture from the canonical list if the nature of architecture has changed, and in this vein Stecker notes a rising tide of building design that is functionally oriented without significant aesthetic investment. A further alternative is to say that architectural objects are all art works, or at least intended as such, within bounds. To make sense of the term “vernacular architecture” requires that we see architectural objects as typically art objects, while we make allowance for a broad class of architectural objects that are low-art rather than high-art.

If the negative view is correct, then we need at least a workable set of criteria by which to discriminate architectural objects as art. To this end, we may draw on our intuitions, norms, or socially expressed views. Further considerations may include the pertinent cultural tradition in which an architectural object is created, whether particular sorts of aesthetic qualities count more towards artwork status, or whether there is instrumental benefit in considering the object as art.

**What renders architecture distinct from other artforms (if it is one).** If architecture counts among the artforms, we may think that it has distinctive features as such. For example, Scruton (1979/2013) suggests that architecture is a _non-representational_ artform since it need not—and generally does not—represent any content. Despite architecture’s non-representational status, Scruton agrees with Langer (1953) (and anticipates Goodman (1985)) that architectural objects can express or refer; in his view, to thoughts associated with expressed properties. Others focus on architecture’s distinctive commitments to creator or user engagement. Winters (2011) sees architecture as a “critical” artform, requiring the creator’s engagement in the environment where other artforms may tolerate a creator’s detached stance. Another distinctive mark of architecture among the artforms is its nontraditional status as a narrative medium: the design of circulatory pathways allows architectural objects to communicate a sequence of events through the movement of visitors or inhabitants.

**Whether architecture includes all built structures.** Among the issues noted here, that of the greatest consequence is the question of what counts among architectural objects. On an _inclusivist_ or expansive conception, architectural objects are those designed objects ranging over the whole of the built environment; on an _exclusivist_ conception, the range describes only some coherent subset of the whole of the built environment. Examples of exclusivist subsets include (a) only built structures that people can occupy (typically: houses, temples, office buildings, factories, etc.), or (b) only built structures designed in primarily aesthetic (rather than purely functional) terms. An inclusivist conception entails a vastly larger architectural domain of objects—and areas of practice and inquiry.

Proponents of exclusivity (S. Davies 1994, Scruton 1979/2013) are eager to protect architecture as the preserve of just those objects in the built environment with abundant, apparent, and appreciable aesthetic qualities; clear creator’s intentions to generate art; or the panache of high art and engagement with art historical trajectories. Stecker (2010) offers a putative variation, allowing that as a broader creative medium architecture has an inclusivist character—though, as an _artform_, architecture is exclusivist. Scruton, for his part, identifies a specific intent to _exalt_: architecture as a pursuit has lofty goals or purposes, such that architectural objects do as well. One commonsense justification for exclusivity overlaps with an institutionalist perspective: laymen and connoisseurs alike can differentiate between the striking work of an architect and the humdrum, cookie-cutter building design of a draftsman.

Arguments for inclusivity include Carlson’s appeal (1999) to consider the class of architectural objects as of a continuum with the broader class of everyday designed objects, where everything admits of possible aesthetic appreciation. Then, _contra_ Stecker, we can effortlessly count all built structures as architecture, though some such things—like garage doors or drainage ditches—will neither look like, nor be, art. Another line of attack is to respond to exclusivists that architects simply have intentions to create objects that are, in one aspect, art—and that they may fail as art is beside the point. Further, it may be that recognizing intentions is irrelevant to judging a built structure as architecture, as when we judge as architecture the vernacular structures of foreign cultures. In response to Scruton’s exaltation criterion, the inclusivist may note that for all artforms, there are typically innumerable objects in the domain with no such goals—and possibly no goals at all. Finally, inclusivism has its own commonsense justification: we standardly refer to a creator of a mundane built structure as the architect, which seems less a linguistic shortcut than recognition of the training and ethos attached to the creator of architectural objects.

It is not clear how to craft intermediary positions between inclusivism and exclusivism, given that the various brands of exclusivism are not absolute and test cases are instead subject to judgment along any number of parameters. Stecker’s exclusivism for only architecture as art represents one such relativized stance. Inclusivism, by contrast—along with any attached views on, for example, architectural appreciation or the nature of aesthetic success in architecture—is an absolutist doctrine. All elements of the built environment—and much else besides—must count as architectural objects, or else the view fails.

3\. Metaphysics
---------------

The metaphysics of architecture covers a surprising range of questions for those who see in architecture no more than metaphysically mundane built structures or stones, wood, metal, and concrete arranged in a pleasing fashion: the nature of architectural objects and their properties and types, the relations of architectural parts and wholes, and the prospect of architectural causality.

### 3.1 Ontology

Given the familiarity of architecture in, and as constitutive of, our physical surroundings, it is strongly intuitive to think of architectural objects simply as buildings, in the way we think of the objects of the sculpture artform as sculptures, or the objects of cutlery as forks, knives, and spoons. Yet such intuitions may be misguided. For one, though some built structures—including roadways, bazaars, and newspaper kiosks—are not buildings _per se_, we may take them to have architectural properties and thereby consider them as architectural objects. For another, the outputs of architecture are not limited to built structures but include as well models, sketches, and plans, and this variety prompts questions as to whether these are all reasonably considered architectural objects and which, if any, such form of output represents a primary sort of object in architecture. A third consideration is the focus in architecture, not solely on whole or individual buildings, but also on parts of buildings and buildings considered in context, among other buildings and in landscapes (downwards and upwards compositionality or modularity). A fourth consideration is that—as with music and photography—where multiple instantiations of a given work are possible, we may dispute whether the work is identical to the instancing built objects or else to the common entity (e.g., plan) on which those instances are modeled. In addition to such challenges, the intuitive view must best alternative views.

**Instantiating architectural objects.** To address one sort of question about the identity of an architectural object, we seek _kind-wise_ criteria that establish when an object is _architectural_, instead of being non-architectural altogether or only derivatively so. To address another sort of identity question, we look for _instance-wise_ criteria that establish when an object is this or that singular object, or an instance of a multiple object. Ready criteria for identifying object instances in architecture include historical, environmental, stylistic, and formal features—all of which may be read as signaling intentions to design particular, self-contained architectural objects. One issue, however, is whether traditional criteria (or others as may be posed) are sufficiently specific so as to skirt vagueness problems that all such artifacts may face (Thomasson 2005) and provide markers of being a _bona fide_ instance of a multiple work or a replica of all other _bona fide_ instances (Goodman 1968/1976).

**Architectural objects as ontologically distinctive.** Yet another way to pick out architectural objects is to set them apart from other art objects or artifacts. Assuming there is more than one art ontology (_see_ Livingston 2013), we might look to define a distinctive architectural ontology by reference to specifically architectural qualities, such as “mass” (degrees of heaviness and lightness) or directedness (in circulatory pathways); or utility considerations, such as functional design, use, and change; or everyday artifactual features. An inclusivist may add features special to the built environment beyond the realm of buildings.

**Kinds of architectural ontologies.** Architecture’s distinctive qualities may help sort among candidate ontologies. One option is concretism, which—in keeping with standard causal efficacy claims and expressed intentions of architects, clients, and users—suggests that architectural objects are either built structures or, on one variant, otherwise physically instantiated designs for such structures (such as models). Concretism is supported by an artifactual ontology that subsumes architectural objects into the class of objects that are the product of intentions, designs, and choices (on the view that _all_ art objects are best so understood, see Dutton 1979, S. Davies 1991, Thomasson 1999, and Levinson 2007.) One version of architectural artifactualism identifies buildings as systems (Handler 1970). As against concretism, intentionality may be the mark of materially constituted, designed architectural objects but that need not commit us to their existence alone or their primacy among such objects. Moreover, taking intentions as determinative leaves the concretist with the problem of shifting intentions and unintended goals attached to built structures over time.

Abstractist alternatives follow a well-worn path in aesthetics (Kivy 1983; Dodd 2007; critics include S. Davies 1991; Trivedi 2002; Kania 2008; D. Davies 2009) and accommodate an expansive architectural domain that includes historical, fantasy, and unbuilt works. Per classic Platonism, abstractism allows identification of an architectural object and concrete counterparts—including multiple replicas—by reference to a single, fixed, and unchanging background source of what real world structures (or fantasy structures) are and should look like.

Against abstractism, some architectural objects are apparently singular because historically and geographically contingent (Ingarden 1962); it is unclear what an experiential account of architectural abstracta looks like; and abstracta are not created whereas architectural objects are. De Clercq (2012) further proposes that, even if there were abstract architectural objects, we don’t refer to them. If I refer to “10 Downing Street”, my expression picks out the built structure, not the plan, nor any other abstract representation or entity to which the built structure at 10 Downing Street corresponds. However, a plausible alternate interpretation of the referent is as “the abstract object physically instantiated by the structure I am perceiving (or have perceived)”.

As an alternative to an abstractist-concretist divide, a pluralist ontology (_per_ Danto 1993), allows “material bases” and “aesthetic ideas” as different sorts of architectural objects. Goodman’s account (1968/1976) lends itself to a pluralist, or at least aspectual, reading. He suggests that, but for certain conditions unmet, an architectural object could be identified as those structures that perfectly realize a corresponding plan or other suitable architectural notation ([see §4](#ArcLanNot)). On his nominalist view, the objects turn out to be the built structures but an available realist interpretation—which may better accommodate the multiples that are key to his story—takes the objects to be the class of such structures.

Another alternative suggests that architecture consists in actions or performances (_per_ Currie 1989; D. Davies 2004), rendering derivative any concrete structures or “traditional” abstract entities. Lopes (2007) proposes the possibility of an events or temporal parts ontology for a kind of built structure that passes in and out of existence, though De Clercq (2008, 2012) counters that such can be rendered in a material objects ontology through temporal indexing. Yet other ontologies are contextual or social constructivist, proposing that architectural objects exist, beyond their status as structured materials, in virtue of ways our reality is framed, psychologically, socially, or culturally (_per_ Hartmann 1953, Margolis 1958). A shift in any such frame may bring about shifting identity in an architectural object, in the manner of Borgesian art indiscernibles (Danto 1964), and it may count in favor of those ontologies that architectural indiscernibles are all around, in the form of repurposed built structures.

Picking an ontology has wide-ranging significance, relative to questions of material constitution, composition, part-whole relations, properties, and relations in architecture, as well as the character of architectural notation, language, cognition, or behavior; there are also ramifications for simplicity and complexity, and the nature of ornament, proportion, context, and style. In architectural practice, the ontology of choice also colors perspectives on such matters as intellectual property rights, collaborative work, and preservation of architectural structures.

### 3.2 Part-Whole Relations

On one customary view of architectural objects, individual built structures (or their abstract counterparts) represent the primary unit of our aesthetic or, for that matter, any architectural, concern; all other ways of carving up the architectural world are derivative. This view is consonant with an equally customary perspective identifying architectural objects with architectural works. Alternative views include a “mereology”, in which parts of built structures (or their abstract counterparts) constitute independent architectural objects; and an environmental contextualism, in which collections of built structures (or their abstract counterparts) constitute independent objects. Both alternatives share a commitment to some form of compositionality among architectural objects, that putting bits together yields aesthetically meaningful and utility-bearing composites, and taking them apart yields like results. In this, the mereological view represents a _downward_-compositionalism, suggesting that architectural aesthetics demands our focus on structural or other elements that can be meaningfully distinguished. Environmental contextualism represents an _upward_-compositionalism, suggesting that architectural aesthetics cannot be pursued entirely separately from the aesthetics of cities or towns (_per_ Scruton 1979/2013).

### 3.3 Causality

Questions about causality may seem out of place in discussions of immobile objects, such as most architecture represents. Yet architectural objects appear to have a role in causing events to happen or other things to come into being. For example, socio-psychological evidence suggests that architectural objects cause behavior, and much of architectural design is predicated on this claim. A further question is whether one architectural object may “cause” another. Thus, the presence of one or more architectural objects might have a causal effect on the genesis or character of one or more further works by dint of social utility, planning needs, or aesthetic drivers. If true, then—as with consequentialism in ethics—further questions arise regarding the range of causal possibilities. Where ethicists ask whether a bad may generate a good, we may ask whether the presence or construction of a functionally or aesthetically impoverished architectural object might occasion the presence or construction of a more useful or pleasing architectural object.

4\. Architectural Language and Notation
---------------------------------------

The notion that there is or should be an architectural language—or more than one—has a provenance dating to ancient times. In classic form, the architecture-as-language thesis runs from the Vitruvian suggestion that the orders present rules for combination and ordering of architectural parts, through Alberti’s rhetoric-inspired model for architectural description (van Eck 2000), and an analogy common among Renaissance and early modern authors (such as Wren) of architectural rules and expressive capacity with the Latin language. Lavin (1992) suggests that Quatremère de Quincy (1803) develops the thesis, from a traditional view of the classical orders as grammatical building blocks directly representing primitive structures, to a modern view of structural elements broadly representing social and moral ideas and principles. Variations of the thesis range over elements of an architectural language, how it may be used, and from whence it may be derived.

The core idea, in its most prominent form, is that architecture as a corpus of design ideas (realized or otherwise) features a set of fundamental design and style elements which can be combined and related according to a set of rules (syntax), capable of constituting or parlaying meaning (semantics), and subject to contextual sensitivity and internal or relational constraints on deployment and realization (pragmatics). Beyond these structural parallels with basic facets of natural language, it is held that the purposes and possibilities of architecture _qua_ language yield further parallels, best explained by the notion that architecture has, or even _is_, a language.

Proponents of such views tend to subscribe, however, to defenses rooted in one or another feature of language. On syntactically-inspired views—the perspectives most indebted to the Vitruvian account—there is at least one architectural grammar or set of rules for guiding proper assemblage of parts and orientation, relation, and combination of whole architectural objects. Some late twentieth century architectural theory embraced a grammar framework (Alexander et al. 1977; Hillier and Hanson 1984); such a view also underwrites a formalist vision for CAAD (Mitchell 1990). Adherents of the view (Summerson 1966) assign themselves the central task of identifying such rules. Even if this is achieved, though, a greater puzzle is whether there are identifiably _preferable_ syntaxes—and what the criteria should look like.

On a semantics-inspired view, architectural objects or their component parts bear meanings. A primary motivation for this view is that, like objects of other artforms, architectural objects are expressive, which suggests that what they express is meaning (Donougho 1987). Proponents point to an array of architectural meanings, internal or external to the object. The former tells us something about the architectural object (its function or internal composition) or how it relates to other architectural objects (stylistic conventions); the latter tells us something about the world, as for example, national or cultural associations (_per_ geographically variant design vocabularies), theological or spiritual significance (_per_ religious design vocabularies), or _per_ Hegel (1826), the Absolute Spirit. A more ambitious proposal (Baird 1969) has it that architectural objects exhibit such semantic phenomena as metaphor, metonymy, or ambiguity.

Goodman (1985) proposes that buildings have meaning in that they function symbolically relative to properties, feelings, or ideas, sometimes through “standard” denotation, as when representing symbolically (whether as building part or whole) some other object in the world. Primarily, though, buildings function symbolically through exemplification (literal or explicit denotation) or expression (metaphorical exemplification) of properties of ideas, sentiments, or objects in the world. Buildings only constitute architecture _per se_, in Goodman’s view, if they bear meaning in one or more of these ways. While Goodman may have identified a denotative role for buildings, this is not clearly a semantic role.

A third approach, rooted in semiotics, emphasizes the role of architectural objects as signs that prompt spectator behavior (Koenig 1964, 1970) or indicate aspects of themselves, such as function (Eco 1968). In either case, architectural objects are taken to operate as communicative systems (Donougho 1987). A semiotics program was embraced by the late 20th century postmodernist movement in architecture (Venturi, Scott Brown, and Izenour, 1972/1977; Jencks 1977).

The architectural language thesis, in its various forms, is widely discredited in recent philosophy of architecture. To begin with, architecture features some qualities and exhibits some phenomena resembling those of natural language, but the parallels are neither comprehensive nor fully compelling. On the syntactic side, architecture may feature some brand of compositionality but different parts of architectural objects do not appear to function as do phrases or clauses (Donougho 1987). Syntactic “wholes” do no better, as there are no architectural assertions (Scruton 1979/2013). As regards semantics, no likely candidate for an architectural vocabulary regularly yields any specific class or instance of meaning. Whereas for Vitruvius the Ionic order connotes female sensibility, it has a scholarly or deliberative meaning for Renaissance architects, “_gravitas_” for Blondel (1675–1683), and good governance for the architects of New York’s City Hall. Nor are there truth conditions such as might supply the meaning of a well-defined architectural sequence (Taurens 2008). It’s not even clear that architectural communication is best understood as _sequential_; Langer (1967), for example, suggests that the symbolic communication of architectural objects is instead holistic. As for pragmatics, there is no clear parallel with implicature or related phenomena, hence architecture is incapable of the accuracy or concision of expression we associate with language (Clarke and Crossley 2000). Finally, relative to semiotics, not all—or even many—buildings signify and we would only _want_ some to do so.

On the bright side, it’s not clear that we should want architecture to be more language-like. Regarding semantics, whatever we gain in fixing particular meanings to architectural objects, we stand to lose in fungibility of their forms. Regarding syntax, adherence to grammars brings the utility of standards and, _per_ Scruton and Harries, a template for the community’s voice—but for some may represent stifling constraints on the aesthetic imagination.

In the end, it is useful to ask what work we expect the architecture-as-language thesis to do. One view (Alexander et al. 1977; Alexander 1979) takes the thesis to underscore the robust nature of design patterns and known design solutions and, for a given culture, a common vocabulary (Donougho 1987). However, it may be sufficient to highlight ways in which architecture is _like_ a language, though they do not add up to an architectural language (Forty 2000). If so, then the thesis works best as a powerful metaphor rather than as literal truth.

An alternate take on linguistic phenomena in architecture appears in Goodman’s proposal (1968) that we think of notational systems or schemes for the arts as symbolic systems with potentially language-like features. Goodman suggests that architecture is a borderline case of an _allographic_ artform, as its notational schemes—in the form of plans—are intended to guarantee that all objects as are compliant are genuine instances of the work. (Said intention, Goodman proposes, is not fulfilled.) That an object instance the work just when compliant with the notation would indicate the notation’s satisfaction of syntactic and semantic criteria providing requisite grounds for identity across instances, and signal a corresponding insignificance of historical context and conditions of production to the object’s identity. Goodman, for his part, balks at taking architecture to be truly allographic given the core role of history and context in generating particular structures, and notational ambiguity that marks the analog medium of traditional plans. Digital design may well resolve the ambiguity problem, however, and allow indexing for history and context, rendering architecture allographic _per_ Goodmanian criteria (S. Fisher 2000b). The term “architectural language” then takes on a different—and more plausible—sense.

5\. Formalism and Anti-formalism
--------------------------------

### 5.1 Formalism

In its most general sense, formalism works in architecture as it does (or doesn’t) in other artforms. Thus, architectural formalism suggests that the sum total of aesthetic properties of an architectural object are or arise from formal properties, such that our aesthetic judgments are warranted based on experience and assessment of just those properties. As architectural objects are typically non-representational and designed with manipulation and relation of forms as a primary task, it is natural that their formal properties be seen as playing a central role in our aesthetic appreciation of them. The question posed to the traditional (“hard”) formalist is whether those properties are _unique_ or at least dominant drivers of aesthetic properties and judgment, a question underlined by important roles of history, styles, and other contexts in our grasp of the architectural enterprise and individual architectural objects. Our aesthetic judgment of I. M. Pei’s Louvre Pyramids is surely to some degree in reaction to their “pure” form but—for the aware spectator—perhaps just as much in reaction to their relationship to historical context (the Giza pyramids as emblematic of pyramidal form in architecture, and of monumental architecture altogether) or setting (in contrast to the ornate neo-Baroque Louvre buildings that surround them, but in keeping with traditional French emphasis on geometric form in design).

Variants of architectural formalism take formal properties as the properties of (or arising from) the material or physical properties of built structures (as consonant with concretism), or as the properties of (or arising from) the total properties specified by a set of formal parameters we identify with the architectural object (as consonant with abstractism). Further architectural strains are characterized by moderation (_per_ Zangwill 2001), suggesting that some architectural objects are best understood by appealing to their formal properties, others not; or by assimilation of canonically non-formal properties to a formalist scheme (in the manner of Levinson’s “indicated structures”; see S. Fisher 2000b); or by a “mereological” view wherein some _parts_ of a given architectural object may be best understood and judged by their formal properties, others not. For the merelogico-formalist, it might count in favor of considering such parts as independent architectural objects that we can judge those parts on a formal basis alone.

Formalism appears in some traditional architectural theories as a normative practical or critical guideline, namely, that our best design thinking takes as central an architectural object’s shape, color, and other formal elements. Other, non-formal aspects of an architectural object are discounted as contributing to its success. Mitrovic (2011, 2013) embraces a normative formalist approach to criticism, on the grounds that the deeply visual nature of much cognition militates against basing appreciation or evaluation of architectural objects exclusively or primarily on features we understand through non-visual means (such as context or history provide).

### 5.2 Anti-Formalism and Functional Beauty

The anti-formalist traditionally focuses on the importance to aesthetic judgment about non-formal properties, including historical context; other, categorial forms of context (Walton 1970); or non-cognitive properties. As an architectural application would have it, we likely judge Jefferson’s University of Virginia campus as stately or dignified or evocative of democratic ideals because of the neo-classical design, the campus’ place in histories of American architecture and university architecture, and its continuous rededication through the everyday functioning of an enduring, living university. None of this judgment appears to have particular roots in _forms_ Jefferson deployed, except as befit a neo-classical style—which style may be best grasped in historicist terms.

Aside from historicism, a principal variant of architectural anti-formalism derives from functional beauty theory, which has its roots in (a) a late modern tradition of judging an object beautiful if fit for its intended function (Parsons and Carlson (2008) find this tradition in Berkeley's _Alciphron_ (1732) and Hume's related suggestion (_Treatise_ (1739-40)) that beauty of artifacts consists in their appearing to bear utility), and (b) Kant’s proposal that architecture is an artform capable of generating dependent beauty. (In the latter case, beauty stands in relation to concepts with which we associate architectural objects, which for such objects are typically the ends towards which they are created.) One modern version proposes gauging the beauty of a designed object by reference to designer’s intent in crafting a functional solution; for S. Davies (2006), where an object displays functional beauty, aesthetic considerations and the object’s primary function each act to shape the other. _Per_ Parsons and Carlson (2008), the problem with such intentionalist accounts in architecture (or elsewhere where functional beauty pertains) is that functions _change_. To work around this difficulty, they suggest, we need a theory focused on “proper functions” for the artifacts in question. This view is modeled on a selected effects account of biological functions, as translated into a marketplace-driven scheme, where evolution of design solutions is driven by demand over time.

Functional beauty faces several challenges. Even in their advocacy, Parsons and Carlson caution against the suggestion that function _solely_ determines form, as that would neglect other features of artifacts not possibly highlighted by their functions. Such features include cultural significance or _aspects_ of non-dependent beauty as may be found in, for example, architectural ornament. (In Davies’ picture, there is no such neglect because the functioning of artifacts—including art and architectural objects—may have a cultural, spiritual, or otherwise non-mechanical cast.) In the architectural realm, another challenge is posed by ruins, which may be beautiful but have no functions. To the charge that these represent counterexamples to functional beauty theory, one tack is to answer that if ruins represent architectural objects, they are dysfunctional and their beauty is manifest in non-functional ways (Parsons and Carlson). Functional beauty theory is saved on the whole but not as universally characteristic of architectural objects.

A further challenge casts doubt on seeing functional beauty as the only variant of dependent beauty, or beauty as the sole aesthetic valence of interest to a viable notion of dependent aesthetic properties. In an architectural vein, those variants may include spiritual, emotional, or conceptual frameworks we bring to our grasp of such built structures as houses of worship, memorials, or triumphal arches. We can tell functional stories about these sorts of structures in sociological or psychological analyses but not (or not only), as functional beauty accounts would have it, in terms of their mechanical or system-wise functioning.

Looking beyond functional beauty—or more broadly, dependent beauty—accounts of architecture, an inclusivist will seek the thread that ties together architectural objects with aesthetic properties of all description, be they functional, otherwise dependent, or freely (independently) endowed with beauty or other such properties. Thus, a modernist gas station and a Tschumi _folie_ may share an elegance unrelated to functional ascription _or_ the lack thereof. A general theory of architectural objects, along inclusivist lines, suggests at least a moderate formalism.

6\. Architectural Experience, Knowledge, and Appreciation
---------------------------------------------------------

### 6.1 Experience of Architecture

A staple of philosophy of art is that our experience of art objects—direct or otherwise—is central to basic belief formation about them (first and foremost, aesthetic belief and appreciation of art objects). The philosophy of architecture is generally in agreement, though architectural objects may be of special character in this regard, as our experiences of them gives rise to or influences an extended range of psychological states. Beyond pleasure in architectural beauty or other “positive” aesthetic properties, experience of built structures also contributes to neutral and less positive states of mind, and shapes how we broadly take in our environment. A piece of that environmental understanding is local to the built structure itself: the ways we experience architectural objects may contribute to how we comprehend, and interact with, those objects.

In addition to facilitating understanding, appreciation, or use of architectural objects, experience might also play—or reflect—a constitutive role. On Scruton’s view, experience constitutes for us the architectural object as an _aesthetic_ object (1979/2013). For Ingarden (1962), architectural experience entails not only our cognitive grasp of the built structure’s physicality but as well our grasp of its designation as a specifically _architectural_ object rather than, say, as an arrangement of bricks that happens to have the structure of a house.

The content and corresponding faculties of architectural experience likely include some mix of the cognitive, emotive, and sensual. Whereas an abstractist may claim that experience of architectural objects is solely a matter of intellectual grasp, even an anti-abstractist formalist needs the sensory as well to account for experience of concrete shapes. Abstractist intellectualism notwithstanding, accounts of architectural experience typically focus on multiple content modalities. Sauchelli (2012a) proposes the use of cognition in grasping pleasure (“intellectual pleasure”) as a central feature of architectural experience. The idea is that fully comprehending the pleasure of the experience and thereby establishing its aesthetic value requires cognition, in the form of attention to details and understanding of the architectural object.

A blend of the cognitive and sensual is also characteristic of Scruton’s proposed “imaginative perception”—the notion that we _may_ perceive the details of built structures in various ways, depending on directions that our imagination takes us. Scruton (1979/2013) takes this cognitive act—reminiscent of seeing-as and free play of the imagination—as crucial to architectural experience. We are at all turns required to make interpretative choices in parsing ambiguous or multiform aspects of the built environment. Scruton focuses on voluntary deployment of the imagination in perception at a macro-level, concerning such matters as whether we see a sequence of columns as grouped one way or another, or see pilasters as ornamental or structural. The voluntary aspect of this account is critical to Scruton’s emphasis on the importance of taste and discrimination to architectural aesthetics. In this, he loosely tracks Geoffrey Scott’s view of architectural experience as “sensuous perception” interpreted through attendant values (1914/1924).

A generalized version of this account looks to perceptual tasks at a more granular level. Our experiences of space and spatial positioning, depth, edge detection, color, and light yield multiple interpretative possibilities across architectural objects, including the simplest forms and smallest or largest parts of objects. These perceptual tasks are pervasive and constant; sometimes involuntary and in the background, and other times as shaped by our willful imagining. If the involuntary tasks also represent or shape interpretative acts, architectural experience by Scruton’s measure is that much less subject to aesthetic taste or discrimination.

The dimensions of architectural experience are even larger when taking into account the full breadth of the sensual. Following a long tradition of viewing architecture through art historical lenses, Scruton focuses on architectural experience as primarily visual and static. In addition, though, other sensory modalities are factors: changes in aesthetic judgments follow changes in those other sensory experiences (Sauchelli 2012a). Such modalities among the non-visual include the tactile, aural, and olfactory. Moreover, much architectural experience is proprioceptive, incorporating visual information into a broader set of stimuli to grasp bodily position and movement in relation to the built environment.

Sensation of movement might seem irrelevant to experiencing an immobile object, save for the fact that, in architecture (as in sculpture) not all facets of a given whole work, or many other architectural objects, can be perceived at the same time. The spectator or user must move around or within the object to perceive any significant percentage of it, much less the whole. Experience of movement around architectural objects highlights for us the design feature of circulatory paths and contributes to grasping an object’s formal features (such as rhythm in spatial patterns) and perhaps, at least derivatively, its aesthetic features (such as somberness) (Sauchelli 2012a; Rasmussen 1959). These aspects of architectural experience capture the immersive nature of a spectator’s or user’s relationship to a built structure. Further aspects of our experience may capture the architectural object’s immersion in _its_ larger environment and surroundings, its “localized nature” or context and “sense of location” (Carlson 1994). As architectural objects standardly shape our actual, imagined, or remembered bodily engagement, so are our richest experiences of architecture informed by such engagement (J. Robinson 2012).

Central as bodily experience may be, it cannot be the only source of architectural beliefs. Considering the great breadth of the architectural enterprise, it may not even be the best source. Other sources include access to beliefs about works through standard representational modes that are not the works themselves, transmission of tacit working knowledge through apprenticeship learning, and collective belief formation through client briefings and studio group critical assessment (“crits”). On these and other bases an architectural knowledge of special character is built.

### 6.2 Architectural Knowledge

Knowledge of a building or other architectural objects follows well-worn paths in some aspects of general knowledge of art. In particular, architectural beliefs encompass judgments of aesthetic properties of the built environment, are norm-governed in some fashion, and may be transmissible via testimony. Yet other aspects of knowing architectural objects diverge from the well-worn path, as reflective of special characteristics of the architectural enterprise and its products and consumption.

One traditional division of architectural knowledge promoted by architectural historians, theorists, and practitioners—and focusing squarely on creator’s knowledge—has it that there are two basic kinds: the theoretical/historical and the practical (J. W. Robinson 2001). _Theoretical_ and _historical_ brands of architectural knowledge encompass viable beliefs about familiar core concerns of architecture, including basic design elements of the built environment; their combinations, relations, and properties; their style; external factors (social, economic, cultural, etc.) which shape design; and historical contexts into which they fit. Some such beliefs are empirically supported; others not. _Practical_ brands of architectural knowledge encompass viable beliefs about the engineering and technical means of constructing architectural objects, ensuring structural integrity, and guaranteeing mechanical function, socially, industrially, or ecologically beneficial use. Such beliefs—particularly as hitched to formalized, experimental, or predictive dimensions of the enterprise—are sometimes seen as constituting an _architectural science_. They are typically (though not exclusively) empirical in character and, to some tastes, relegated to a status of _adjunct_ architectural knowledge, that is, useful for architecture but outside the domain proper. What counts as practical knowledge in architecture is often seen as encompassing beliefs of a largely non-aesthetic nature.

Yet other categories reflect a range of types and sources of architectural knowledge. Another division distinguishes between architectural beliefs associated with creators and users. My experience of a built structure _qua_ creator is perforce different than my experience of the same structure _qua_ user, and the sorts of beliefs I arrive at may differ accordingly. As architect, Jones believes that an arch of one design but not another will keep the bridge up; as someone strolling underneath the bridge, Smith believes that an arch of a different design would have been a greater aesthetic success. This much accords with other artforms featuring practical functions. Further, architectural beliefs may differ by their technical or non-technical nature; by perspective and role of the belief-holder; or by facts about physical experience of the work or other modalities of belief acquisition.

**Architectural knowledge in broader context.** To see how architectural knowledge may be similar to, or differ from, aesthetic knowledge generally, consider two dimensions of aesthetic knowledge, knowing _through_ art and knowing _about_ art (Kieran and Lopes 2006). As concerns knowing _through_ architecture, cognitive content arises in reflecting—to varying degrees—taste and style sensibilities of its creators, structural properties _per_ engineering principles deployed; and cultural and social values of historical, communal, and economic contexts. To know a built structure in this regard is to know such matters as the tradition in which it is built; design aspirations of the architect and initial occupants; and intentions relative to contributing to the built or natural landscape. The success of this thesis is predicated on successful communication through architectural objects, whether as symbols or otherwise.

Architectural belief and knowledge have as well wholly distinctive features, reflective of special characteristics of the domain, its practice, and its objects. These include:

_Beliefs about systems._ Architectural objects (as wholes) are systems or system-like, in that they constitute sets of interrelated structural components, with characteristic behavior or processes yielding outputs from inputs, and where the parts are connected by distinctive structural and behavioral relations (Boyce 1969). That we take whole architectural objects to be (or to be represented as) systems or system-like suggests how architectural beliefs are distinctive among beliefs about artworks. In particular, we have beliefs about architectural objects that reflect functions and interactions of (1) components individually and as parts of systemic wholes, (2) systems as parts of broader environmental contexts, and (3) persons’ behaviors within the systems. Whereas the first two functional and interactional features are typical to all design, the third feature marks architecture as an artform that, in providing an immersive and systemic physical environment, intensely draws on and shapes social, psychological, and economic features of experience. Our beliefs about architectural objects and interactions with and in them are shaped correspondingly, in ways that do not arise in engagement with other artforms.

_Partial and full information._ Representation in architecture encompasses multiple modes, including built objects, physical models, virtual models, data arrays, plans, sketches, photographs, and drawings. Each such mode may be _viable_ as representing an architectural object just in case _some_ features of the object are adequately, accurately, regularly, and optimally represented through the mode. This view of viable representation in architecture is at odds with the standards for such in other artforms. Consider a representation of the Mona Lisa. If you do not have complete visual access through the representation (from any acceptable angle) to the full tableau, you may be said to lack full acquaintance with the work through the representation, and your consequent aesthetic beliefs about the Mona Lisa may be discounted accordingly. By contrast, if architectural beliefs required anything like _full_ acquaintance with the object or _fully informed_ testimony to be viable, our architectural beliefs would not typically or frequently be viable. Rare is the case where acquaintance is full or testimony fully informed—even among those whom we might expect to have the greatest acquaintance, such as a built structure’s architect or developer.

_Socially constructed knowledge._ In architecture, as in other design fields, design problems are not thoroughly or fully articulated all at once or by any particular individual. The primary components of design knowledge—problems and their possible solutions—are instead distributed across persons. This fact about architectural production—and, to a degree, its use—suggests that beliefs we form about architectural objects are formed amidst, and influenced by, such social relations ([see §8.1](#SocConFeaArc)). Art and architecture worlds _per se_ are undoubtedly not a sole source of epistemic norms. Yet social relations and circumstances among architecture’s stakeholders present constitutive conditions for a wide swath of architectural beliefs, suggesting at least a moderate social constructivism.

It may be thought that qualities of architecture such as systematicity and the deeply social character of the discipline are immaterial to _aesthetic_ beliefs. However, architecture is a holistic enterprise: a design decision to cantilever a terrace is at once of aesthetic and engineering significance. In like fashion, that architectural objects constitute systems is pertinent in shaping aesthetic beliefs because there are more and less attractive ways to shape the flow of persons, or even electricity, through a built structure. And that architectural objects are designed through social processes has import for corresponding aesthetic beliefs. For example, aesthetic properties of a given design are subject to crits, the very purpose of which is to influence the creator’s further aesthetic beliefs about the same design.

### 6.3 Architectural Appreciation

As distinct from mere _experience_ of architectural objects, _appreciation_ of architectural objects brings to bear cognition and other inputs, such as history and context. Appreciation goes beyond _knowledge_, too, insofar as we may know an architectural object and its qualities without appreciating it. Thus, Winters (2007) proposes that appreciating architecture consists in _enjoyment_ of architectural objects (from experience, _tout court_), as wed to _understanding_ them, where the latter consists in grasping their aesthetic significance in specifically visual fashion, and critically assessing the judgment of architects in addressing design challenges. Architectural appreciation may be built on the judgment of others; it is essential to _rendering_ judgment. Accordingly, learning to appreciate architectural objects is a cornerstone of architectural education. A key contributing feature in this last regard is acquiring agility with classifying in the domain (Leder et al. 2004).

The appreciation and judgment of architectural objects are typically thought to reflect aesthetic _and_ utility-wise considerations, and engage individual perspective, experience, reasoning, and reflection such as we associate with appreciating and judging in other artforms. Drivers in appreciation and judgment _special_ to architecture include social framing and environmental psychological factors that are a consequence of architecture’s intensely public nature.

**Exceptionalism.** One question regarding appreciation is whether there is a special mode attached to architectural objects. We might think this is so given that, unlike most arts (though very like other design forms), appreciation in architecture is aesthetic _and_ utility-oriented. A resulting puzzle is whether, and under what circumstances, we might have one without the other. We might think as much if, say, it is possible to appreciate the Roman Coliseum’s stately and antique features without any appreciation of its intended function or actual use. On the other hand, a view of appreciation embracing functional beauty theory may suggest that the Coliseum’s functional and free beauty are on at least equal footing—or that the Coliseum’s stately aesthetic and its role as amphitheater for staging spectacles (for example) cannot be separated altogether.

Further questions regarding appreciation concern the relative roles in appreciation of individual experience of architecture, as against the social or the environmental.

**Individual Appreciation.** The prevailing philosophical view of architectural appreciation is a psychological account with debts to the Kantian tradition: direct, immediate aesthetic experiences of architectural objects among individuals constitute the basis of appreciation. (Iseminger 1981 provides a general aesthetics account in this vein.) A primary variant has it that architectural appreciation is the product of individual cognition of the content, form, properties, and relations of architectural objects. A recent variation suggests that, in addition to (or _in lieu_ of) cognitive response, physiological experience (proprioception) is a central source of beliefs associated with architectural appreciation. On either model, it is experience of individuals that feeds and influences appreciation.

**Social and Environmental Role in Architectural Appreciation.** Direct, immediate individual experience is not the only source of information shaping architectural appreciation. Considering the breadth of the architectural enterprise, it may not even be the best source. Others include access to information about works through standard representational modes that are not the works themselves (for example, drawings or photographs), transmission of tacit working knowledge through apprenticeship learning, and collective belief formation through client briefings and studio crits.

Architectural appreciation is _social_ in building on our understanding of architectural objects as it develops, and matures, in experience of a built structure with and in relation to other individuals and groups of people. It is also social in that we learn markers of appreciation among those with whom we share such experiences or (_per_ Scruton 1979/2013) imagine ourselves to do so. Indeed, a central goal of architectural education is structured imparting of collective wisdom as to how to best classify architectural objects and, relatedly, what the markers of appreciation have looked like, or should look like—as well as how they articulate with practical knowledge.

Further, architectural appreciation is _environmental_ in building on our understanding of architectural objects based on experiences in relation to their natural and built surroundings. On one view, an architectural object may be more difficult to appreciate if we find that relation unexpected, or contrary to normative sensibility (Carlson 1999). If, however, appreciation does not require enjoyment or satisfaction of any sort—and instead engages our understanding of, for example, what was intended and why—we may well appreciate in its own right an architectural object that has a surprising, even obnoxious relation to its surroundings.

7\. Architectural Ethics
------------------------

Some problems of architectural ethics are characteristic of a range of typical moral dilemmas—agent-centered, norms-oriented concerns—as may arise for architects. In addition to a traditional set of questions applied to the architectural domain, architectural ethics also addresses problems special to the discipline and practice—as shaped by its social, public, practical, and artistic nature.

As conceptually prior to a normative ethics of architectural practice, a meta-ethics of architecture assesses alternate ethical modalities, such as whether architecture might be considered moral or immoral relative to its objects (built structures) or to its practices as a set of institutions or social phenomena. Another meta-ethical issue concerns whether moralism or autonomism best characterizes the relationship of aesthetics to ethics, as that plays out in architecture.

**Ethical modalities of architecture.** There are three typical candidate modalities of ethics in architecture. For one, there is the establishment of criteria for ethical norms of the enterprise such as architects in practice may observe. For example, architects can craft designs in ways that lower the likelihood of cost overruns and enhance safety. In an interpersonal vein, architects can represent their work honestly to clients or contractors. Another modality—beyond enterprise-defined ethical norms—is pursuit of criteria to gauge architects as moral agents broadly producing or doing good or bad in the world. For example, architects may create objects that uplift or constrain individual users and inhabitants; other architects may promote social utility by designing housing for those in need of shelter. Finally, there is the modality of seeking criteria to judge architectural objects as morally good or bad insofar as they _directly_ produce pleasure or pain. As an _indirect_ example, a hospital design is intended to facilitate the minimizing of pain, by fostering environmental conditions conducive to excellence in health care and patient well-being. As a more direct case, a bus shelter is intended to reduce exposure to the elements and corresponding discomfort. This last candidate may be attractive if we see architecture primarily as a product rather than as a practice; it is noxious if we are unwilling to assign moral values to artifacts as we do to actions or their properties. A built structure might be inhumane in that it is bleak or uninhabitable, though it does not follow that the structure itself bears inhumane values.

**Value Interaction.** Vitruvian principles underlying much of architectural theory suggest a tendency to link the aesthetic and the utility-promoting. So, too, functional beauty theory recommends that aesthetic and ethical considerations are linked in architecture. To crystallize the matter, we may ask if it is possible for a built structure to be good though not aesthetically so. The question as to how aesthetic and ethical value might be related is a subject of broader concern, with a “moralist” stance that says the two sorts of value are or should be connected (Carroll 1996, Gaut 1998) and an “autonomist” stance that takes the two as (or better off as) independent (Anderson and Dean 1998, Kieran 2001).

In this debate, architecture would seem a promising domain in which to find robust relations. At a moralist extreme, there is the suggestion—supported by some traditions in architectural theory (Pugin 1841, Ruskin 1849)—that aesthetic tasks in architecture simply _are_ ethical tasks, reflecting ethical choices. One prominent moralist perspective locates the ethical element of aesthetic architectural choice in obligations to a sort of honesty, in designing works that accurately represent underlying structural principles or operational capacities. Harries (1997) and Scruton (1979/2013) arrive at similar ethical commitments to architectural design as expressive in a particular fashion, though of shared community values, rather than of function or structural features. From another angle, moralists point to the emotional impact of built environments as indicative of a union of the aesthetically gripping and morally compelling (Ginsburg 1994), though it may be noted that even where we detect such a union we need not judge the aesthetics of the architectural object on the basis of any ethical import so communicated.

At the other extreme, autonomists propose that problems of ethics and aesthetics neither need arise at once, nor need be resolved at once, in architectural design. If we see a correlation in some architectural objects of ethically and aesthetically compelling design solutions, we see in other objects no correlation at all. Thus, the modern city is replete with instances of built structures that are well-functioning, high-utility, and facilitate much good in the world yet meet no one’s standards of aesthetically worthy design. There is good reason to uncouple these values just in case they _must_ conflict. Suppose there is an ethical premium, for example, on the need to create environmentally sustaining structures, and that we identify resolutions of that problem as generally bearing the greatest mark of moral worth. Suppose further that crafting the optimal moral solution (_vis-à-vis_ the environment) always generates the most unattractive design—and that the inverse holds as well (the better the design aesthetically, the worse the design for the environment). Then a connection between ethics and aesthetics in architecture seems improbable.

A third position altogether proposes a pluralism. Sometimes ethical and aesthetic value march hand-in-hand, other times not—and their ways of matching up are diverse and run in various directions. So one architectural design may be aesthetically compelling as it reflects its ethically upstanding character, whereas another design may be aesthetically compelling as it reflects its ethically deficient character. What is required for cases of this last sort is that the evaluator can identify the aesthetic success as grounded, _per_ functional beauty theory, in effective expression of the structure’s ethically deficient function (Sauchelli 2012b). An even more generalized pluralism would suggest that a wide range of aesthetic and ethical valences can be matched up in different ways; we might value a war memorial for the way it grimly expresses the horrors of war.

**Traditional questions of architectural ethics.** One reason that architectural ethics is central to philosophy of architecture is that architects’ actions bear great influence over other people’s moral lives. Architects design structures and environments for people, with concomitant effects on personal behavior, capacity to choose courses of action, and ability to satisfy preferences, visit harm, generate benefit, or exercise rights. As architects’ acts shape spaces, boundaries, and pathways that structure individual behaviors and social acts, they prompt normative ethical exploration along traditional lines.

To begin with, a traditional architectural ethics requires an account of architectural _responsibilities._ Any such account should outline architects’ obligations to other persons, ethical standards on which such obligations may be based, how to ensure such standards might be met, and any other sorts of obligations architects might have, as for example, to historic preservation or environmental protection. As concerns obligations to persons, the range of stakeholders in architecture is great, hence ethical responsibility is diffuse.

A further set of questions concerns _rights_. It is relatively novel to speak of authorial or community rights in architecture; owner or client rights are historically parasitic on property or sovereign rights. Other possibilities include rights of developers, builders, engineers, environments, and societies. As that list grows, two further questions concern the sorts of rights that can be attributed to such parties or entities, and the criteria for distributing and prioritizing them given aesthetic as well as moral considerations.

Architectural _utility_ is familiar as a Vitruvian concept but has a wholly other sense in an agent-centered normative ethics, with a possible moral weighting not found in classic architectural theory_._ Guidelines are needed to determine the usefulness of architectural goods such as built structures, restorations, reconstructions, or plans. These might include their social character, or individual preferences of the architect, owner, end-users, or public. A utilitarian approach to architectural ethics is attractive in capturing the aims of architecture to promote well-being, and relying on a ready marker of architectural value. However, it also discounts other traditional architectural imperatives such as a Vitruvian-style pluralist may honor, including beauty and structural integrity (Spector 2001).

Finally, a traditionalist picture of architectural ethics requires an account of _virtues_ in the domain (though these may be orthogonal to normative ethics). Here is potential consonance with the Vitruvian tradition (and similarly virtue-oriented non-Western traditions)—if we see the “good”, morally educated architect’s virtue and character as our best guarantee of proper and productive weighting of values under differing circumstances (Spector 2001).

**Future-Focused Architectural Ethics.** The focus of ethical rights and responsibilities in architecture is typically taken as relative to present or past. Thus, we speak of obligations to design and build in ethically responsible fashion, or preserve past architectural objects. There are future-focused obligations, as well. Sustainable design is forward-looking even as it is centered on what we design and build today. Further ethical issues may arise relative to future architectural objects. As to obligations to future architectural _objects_, we see as much in the short-term instance of planning around near-future buildings. More puzzling is whether we might have “long-haul” future-facing obligations—apart from utility or prudential considerations—in planning, for example, new cities or accommodations to climate change.

**Special ethical questions of architecture.** Architectural practice generates special moral issues as befit its proper, idiosyncratic features, distinctive among the arts, the professions, and social practices. Most notably, an array of ethical issues of social import arises from architecture’s commitments to creation of a socially beneficial and functional art. Such issues include the nature of “better” housing (and under varying conditions), what (if anything) makes housing an obligation, and ways that such an obligation may accrue to, or be satisfied by, architects. Yet other ethical issues special to architecture range over matters of personal and social spaces and the articulations thereof, including criteria for designing around concerns related to privacy, accessibility (for the public generally and handicapped in particular), respecting community and neighborly preferences, and promoting civic values.

Other ethical matters special to architecture are particularly visible in global perspective. For example, there is inequitable distribution of housing across the developed and developing nations, and part of the solution may be architectural (Caicco 2005). Further, architecture incurs special environmental obligations given that waste and degradation affect, and are affected by, architectural design. One conceptual challenge of sustainability facing architects is to determine whether development is, in principle, a countervailing interest. This is to ask, once environmental obligations are defined, how they may be factored into or weighed against infrastructural and design interests and preferences.

**Professional Ethics.** Architectural professional ethics focuses on architects’ moral choices in the context of practice (Wasserman, Sullivan, and Palermo 2000; T. Fisher 2010). Professional ethical codes govern conduct in (and thereby protect) the architectural profession and avert problems related to business, fiduciary, insurance, or liability functions; the design function is an ethical focus relative to disability. Architectural law highlights professional ethics matters as concern property, liability, and honesty. The law clarifies responsibilities among parties to architectural practice; defines who or what in commercial architectural interactions has moral agency—hence rights; and describes utility-wise or financial measures of distribution in architecture. Legal regulations and judgments prompt conceptual questions regarding such issues as intellectual property in architecture, architects as arbitrators, and architects’ responsibilities to others (S. Fisher 2000a).

_Intellectual Property_. One conceptual issue concerning architectural intellectual property is how such rights are to be weighed against other sorts of property rights, such as domestic or commercial rights. A further issue is determined on the basis of judging architecture to be a service or product. Taking architecture as _service_ means that architects do not have a stake on copyright, as they would then be creators-by-contract; tradition has it that rights to expression of ideas so created accrue to the contracting party. Taking architecture as _product_ supports architects’ claim to copyright, given that expression is their creation—whatever services are performed (Greenstreet 1998; Cushman and Hedemann 1995). Copyright raises other concerns. Even as the law may license creation of an architectural object reflecting core design aspects found in another object created by a different architect, we may find morally blameworthy any cognizant “borrowing” without attribution or permission. Alternatively, we might view this as a routine episode in the history of architectural copying without attribution or permission. The challenge is to define relevant obligations of one architect to others, present or past.

_Architect as judge in owner-contractor disputes_. Architects have a dual role, serving as designer and administrator of architectural projects, and in this capacity may adjudicate between owner and contractor in matters of dispute. Standard issues concern conflicts of interest, grounds for adjudication, and criteria of fairness. Ethical issues of a particularly _architectural_ stripe include the degree to which specifications are poorly satisfied such as warrants reckoning; maintenance of fealty to owners’ interests alongside fairness to contractors and to satisfactory realization of one’s own design; and identification of virtues appropriate to judgment in design-related disputes, along with facets of being an architect that promote (or limit) such virtues.

_Responsibility to others’ design_. Architectural objects often develop over time in cumulative and mutable fashion, through additions and alterations that—perhaps more frequently than not—change the design of a different, original architect (or that of a prior alteration). For any particular changes, or in consideration of design changes overall, we may stipulate obligations to respect original or prior intent and execution. One brand of such obligations, recognized in historic preservation and landmark laws, requires that aesthetic concerns in the public interest trump private interests. Key conceptual questions concern how to determine the source and conditions of any such obligations—and the sorts of responsibilities architects should have to existing structures. Those responsibilities may extend to commitment to the integrity of work by fellow architects.

8\. Architecture and Social and Political Philosophy
----------------------------------------------------

### 8.1 Socially Constitutive Features of Architecture

While all artforms admit of a certain social character, architecture enjoys a particularly social nature, and may even be said to be an _intrinsically_ social artform. There are two prominent candidate reasons as to why this is so. For one, a central aim of architecture is to design shelter and so meet a variety of social needs. For another, architecture as practice is a social process or activity as it engages people in interpersonal relations of a social cast.

The first candidate reason stands or falls on whether, in fulfilling social needs, architecture is thereby rendered a social art. For an artform to be intrinsically social, any such need fulfilled should be critical rather than discretionary or extravagant. Thus, for example, addressing housing demands overall meets the criticality test—though addressing design demands for a third home does not. The first reason looks right because architects often integrate social needs into design thinking. Armed with socially minded intentions, they create built structures which serve myriad social ends. A difficulty arises, however, in consistently upholding such intentions as a mark of the social if (a) such intentions are unclear from experiencing architectural objects, instantiations, or representations thereof, (b) built structures are repurposed, or (c) there are architectural objects with no corresponding relevant intentions. (As an instance, we might consider “found” architectural objects like inhabited caves.)

A second candidate reason that architecture is a social art is that processes of making architecture are thoroughly and ineluctably social phenomena, constituted by interactions of social groupings created and governed by social conventions and arrangements. On this view, the social nature of architecture consists in the status of the discipline as shaped by social convention—where such convention is designated by, and guides actions of, architects and other relevant agents. (This reason directly links the social nature of architecture to its social impact—hence to sociology of architecture; [see §8.2](#SocEffFeaArc).) Architectural phenomena are social, then, because they occur as a result of contracts, meetings, firms, charettes, crits, juries, projects, competitions, exhibitions, partnerships, professional organizations, negotiations, workflow organization, division of labor, and myriad other conventional and agreement-bound purposive actions and groupings of architects and other architectural stakeholders. One might object that, on an institutional theory, _all_ artforms are social in just these ways. However, as played out in art worlds, institutional theories tell us what counts as an art object rather than how such objects are constituted to begin with. Proponents of this candidate must rule out a _naïve architecture_ scenario, the possibility of a lone architect who has no socially significant engagements such as shape her creations.

Either view is temporally sensitive. These parameters and how they constitute architecture’s sociality will change over time, along with vicissitudes in social needs, conventions, and relations.

### 8.2 Socially Efficacious Features on, and of, Architecture

Architecture as object and pursuit produces a great range of effects on social structures and phenomena, in particularly acute fashion in relation to housing, land use, and urban planning. In turn, architecture is shaped by such social concerns as scarcity, justice, and social relations and obligations. Some of this shaping results from social group and institution requirements for space and the structured organization thereof, to promote group or institutional function and identity (Halbwachs 1938). In addition, other social requirements stem from architecture’s roles in meeting concerns and needs for society as a whole.

**Causal direction.** We might see social forces as primarily shaping architecture or else architecture as primarily shaping social forces. Proponents of architecture as “shaper” suggest that architecture provides a means of organizing society, a core Modernist claim but also a thesis of broader currency. Detractors counter that we cannot shape society through the built environment—or we ought not do so. What rests on directionality is how we parse not only theoretical relations but also practical consequences and perspectives concerning a host of social phenomena. To take one example, how we gauge and address the possibilities that architecture offers relative to social inequality is likely a function of whether architecture contributes to, or instead reflects, social classes and social hierarchies. We might wonder whether architects can design so as to promote class equality—or solidarity, justice, autonomy, or other social phenomena as we might foster.

On a third, holistic option, causality runs in both directions. Two examples of such are (a) systems analyses, which take built structures as _social systems_ that contribute to social function, and (b) urban sociology, which takes the city _en gros_ as social structuring of space which shapes its habitants, who in turn shape the city (Simmel 1903). As expanded to _environmental_ sociology, the suggestion is that built environments promote patterns of living, working, shopping, and otherwise conducting commerce among groups and in relation to other individuals.

Other social science domains suggest attendant conceptual issues. For example, one sociological perspective takes relations of individuals and groups to built structures to be analogous to consumption (Essbach 2004); we may ask whether architectural objects enjoy a reception of this particular social sort, a range of such sorts (perhaps as context-dependent), or perhaps, on an individualist read, no sort we would call “social”. For another, sociology of architecture also studies the profession: the backgrounds and relations of architects and other stakeholders, norms governing behavior, and social structures of an architecture world constitute a species of artworld. This last suggestion prompts the question as to what influence we should attribute to an architecture world on the status of architectural objects. The _architecture_ world raises issues beyond those motivating Danto or Dickie, engaging many parties whose interests and preferences are not primarily aesthetic or even economic but driven by social, commercial, engineering, planning, and various other factors. For a third, a Science and Technology Studies perspective (Gieryn 2002) investigates how architecture—primarily in its optimization focus, _qua_ engineered technology—shapes knowledge formation (for example, in laboratory or university design) and organizes social behavior (for example, in architecture for tourism or retail sales). Conceptual issues here include whether there are _global_ principles of optimization of architectural design for social advancement, and what sorts of moral constraints are appropriate to such optimization.

### 8.3 Architecture and the Political

That architecture has _some_ political aspects is a widely held, if not entirely uncontested thesis, with weaker and stronger variants. One weak version suggests that designing built structures entails political engagement through interactions of architects and the public. For example, architects solicit political support of government officials for development projects, governments engage architects to design built structures that express political programmatic messages, and citizens do political battle amongst themselves over architectural designs or preservation decisions. A stronger version highlights a possible role for architecture as an instrument of politics. Thus, Sparshott (1994) characterizes architecture as “…above all the coercive organization of social space”. In other words, designing built structures entails political engagement through the control by force of behaviors and attitudes of people who interact with those structures.

That architecture might have any significant role in politics, or the other way around, calls for explanation. One account stresses that the two domains are oriented around utility-maximization. Utility criteria deployed to judge the worth of architectural objects are exemplary subjects of democratic debate, policy analysis, or community consensus. The appeal to utility is an advocacy strategy for architectural design that dates to Bentham’s Panopticon (1787) or before. Further, traditional architectural promotion of urban and social planning may be linked to social utility criteria for architectural quality; that relationship might run in either direction. In distinct progressive and utopian traditions in architectural thought (Eaton 2002), advancement of social utility is a central motivation in architectural attempts to realize idealistic visions of modes of living and societal organization. (For a critique, see Harries 1997.)

**Power, control, and change.** Architecture’s political cast is also manifest in use as a means of social control. This is not an obvious use in societies where individuals freely choose dwellings or other structures with which they interact. The less choice available in this regard, the greater the possibility of defining people’s choices (a) concerning the built environment they occupy and (b) as a function of that environment. Prominent such architectural types include prisons and refugee camps. Some see potential in architecture for more globally promoting maintenance of power through behavior regulating norms that such built structures _represent_ (Foucault 1975).

Even in generally free or open social settings, though, at the level of urban planning architecture indirectly determines behavior in politically shaped ways. Architects and others planning urban or other densely settled environments take into account such political aims as honoring community values, promoting civic virtues, maximizing social utility, fulfilling professional or public responsibilities, and respecting citizen or leadership preferences (Haldane 1990, Paden 2001, Thompson 2012). The politically hued results of such planning and design efforts, whether pursued in authoritative, consultative, or participatory processes, are architectural objects that change, encourage, or reward particular behaviors.

**Ideology and agency.** Architecture is also used to promote political views, culture, or control, by conveying symbolic messages of power, nationalism, liberation, cooperation, justice, or other political themes or notions (Wren c1670s). One issue concerns the architect’s agency in promoting an official political ideology. In government commissions, the architect generally cedes design control, at a certain point, to the government. Yet the architect is the creator of record. This leaves open whether architects so engaged are promoting the given ideology—or else merely acting as proxies for such promotion. It may seem odd to suggest that, from an aesthetic standpoint the design is of architect _X_ but from a political standpoint the same design is not attributable to _X_.

Political agency among architects is a special version of the more general issue of architect agency relative to clients, including as well corporate and individual clients. Architects have obligations to the client’s aesthetic and utility concerns, and in virtue of those obligations, the responsibility, blame, and praise for a given architectural object cannot be wholly attached to the architect alone. One question is what scenarios or conditions would need to pertain to justify apportioning more or less agency—and, correspondingly, political or moral responsibility—to the architect or to the client, in design and build phases of realizing an architectural object. The phases matter. The design phase appears, at least initially, to be the agency-wise province of the architect, and any post-build phase appears to be generally the province of the client and any relevant user-base (until any such renovation or repurposing as may occur). What happens in phases _en route_ to post-build is murkier, though.

9\. Further Issues in Philosophy of Architecture
------------------------------------------------

**Architectural Failure.** Failed architecture is not a straightforward subspecies of failed art or failed artifacts. Architectural objects may rate as aesthetic disasters yet in some overall sense as successes, unlike non-architecture art objects. And architectural objects may cease to function—or never have functioned at all—yet count as overall successes, unlike a range of (though not all) non-architecture artifacts. Another feature of architectural failure—in keeping with general design phenomena—is that architectural objects may count as successes or failures depending on different states of affairs, context, or remarkably small differences. Thus, a given architectural object may be a failure as an active and integral built structure but not as a ruin (or _vice-versa_). This suggests that background intentions may matter at one early stage, and less so at later stages in the life of a built structure—and that failure may have one criterion for architectural abstracta and other criteria for counterpart concreta. Further, among architectural objects with standard, closely related variants, some may fail while others succeed—perhaps because of a minor distinction such as a garishly painted exterior. A viable account of architectural failure accommodates such features or else devolves failure to the level of some single dimension of architectural objects, such as their putative nature as art objects (failed or otherwise).

**Corruption, Ruins, and Preservation.** Architectural objects as physically instantiated are corrupted or fall apart over time, and may develop new forms in disrepair or as ruins. From an inclusivist, concretist standpoint, a ruin is not any lesser an architectural object than its corresponding newly built structure. An inclusivism is available to the abstractist, too, though she will not see them as the same object—and will rate them both as somehow lesser than the originary object. If we take them as the same architectural objects, we need an account as to how they relate to one another—apparently not by reference to intentions. Even if an architect designed a path to a ruin state, the actual ruin-state would likely take on a wholly different shape. Some may take this as an argument against inclusivism.

Architects typically embrace the Vitruvian premium on _firmitas_ and reasonably assume that built objects should endure—and that they serve intended functions for as long as is desirable. That pair of assumptions in design thinking is at odds with concretism, given corruption and decay of physical constructions as well as routine repurposing in the lives of built structures. The first assumption is consistent with an abstractist vision of everlasting architectural objects. Endurance of serving intended functions is another story: for architectural abstracta, stipulation of repurposing may not change the nature of a given, selfsame object. Depending on degrees of stringency in defining an object’s parameters, corresponding changes in design may bring about an entirely new object or nearby counterpart.

Corruption brings not only total destruction and absence of previously intact built structures, but also enduring ruins or flawed, damaged structures. There is a longstanding premium on ruins in architectural culture as promoting historical perspective, nostalgia, and at least one style (Romanticism). Yet ruins fit awkwardly, if at all, into standard architectural ontologies. The cultural premium is hard to explain for the abstractist, for whom ruins represent defective physical instantiations, which are already substandard in the abstractist worldview. They are even harder to match up with a concretist account as there aren’t underlying creative intentions to build (except in ironic or kitsch building of “new ruins”). Corresponding intentions instead typically concern preservation, restoration, or elimination. Alternatively, Bicknell (2014) suggests we think of ruins as part-objects that, along with full-blown “architectural ghosts”, represent past objects in their intact, built glory and as corresponded to creator intent.

Preservation and conservation possibilities prompt additional considerations, such as whether restoration or maintenance of a built structure sustains it as an authentic architectural whole—and if this is independent of functional integrity, or holds for wholesale reconstructions (Wicks 1994); what conditions warrant preserving or conserving a built structure; and what principles guide warranted alterations or completions of built structures—and whether other considerations may include creativity, fancy, or sensitivity to contemporary needs and context (Capdevila-Werning 2013). As concerns completing unfinished structures, one issue is whether it is possible to discern original design intent altogether. Taken together with contemporary norms that shape our understanding of past architecture (Spector 2001), preservation and conservation are at least partly bound to present-day design conceptions.

**Built Versus Natural Environment.** We typically take built and natural environments to be clearly distinct. This distinction does at least two kinds of work in philosophy of architecture. For one, it helps establish what sorts of things we would discount as architecture even on an inclusivist conception—and even there, we might accept lived-in caves as _found_ architecture but reject most other elements of the natural environment as non-architecture (because neither built nor found). For another, we get a defined sense of natural contexts into which built environments fit (or not), without which any such notions of fit are incoherent. If this is a viable (and desirable) distinction, it is perhaps less clear in what it consists. One candidate view is that we may distinguish the kinds of environments by their different sorts of objects and properties: we find columns in built environments and trees in natural environments and never the other way around. Alternatives highlight the ascribable functions and intent that mark built environments but not natural environments; or different sorts of behavior and obligations attached to the two kinds of environments. While an artifact-centered view of architecture weighs in favor of functions and intent as the most relevant distinction, decaying value of design intent over the life of a built structure may give pause.

**Human and Non-Human Architecture.** What we typically refer to as “architecture” is _human_ architecture. This raises the question as to whether human architecture is assimilable to a larger class of animal-built structures. That would suggest, in turn, that we could assess human architecture—including settlement patterns, individual structures, and built community environments—in ecological, animal behavioral, and evolutionary terms (Hansell 2007). If these are fundamental vantage points for understanding human architecture, that would suggest a need to translate all going accounts—whether focused on aesthetics, utility, or other concerns—into corresponding biological terms. One way to resist this move is to mark human architecture as a particularly human endeavor and creation—likely by reference to intentionality, as flows into aesthetic focus. However, this may just forestall the question as to how to account for _that_ wrinkle—a particularly talented animal builder with notable design intent—in the larger story of animal builders most of whom have less or no such intent.

**Environmental Psychology as Magic Pill.** Another scientific challenge to traditional philosophy of architecture emerges in environmental psychology, which identifies ways that environmental factors such as color, shape, light, and circulatory pattern shape our visual reactions and behavioral patterns within and around the built environment. From such empirical insights, we can fashion constraints on architectural design principles that guide architectural creation, and devise corresponding solutions to particular design problems. As architects learn to exploit this information to advance design, we may ask whether an architectural object may be optimized by the lights of environmental psychology yet—and even _consequently_—deficient in some other, architecturally central respect. Satisfying an architectural object’s intended function or maximizing its utility may well include, or be advanced by, keen attention to environmental factors that influence attitudes to and use of that object. By contrast, moral or aesthetic deficiencies where environmental conditions are optimal seem real possibilities. A resulting question is whether, and to what degree, we could or should abandon moral or aesthetic drivers in architectural design if in the future we could design architectural objects to optimize environmental factors and so meet cognitive and emotional needs, thereby enhancing an architectural object’s reception but at moral or aesthetic expense.
